modules/logging-loki-gui-install.adoc

```diff
@@ -35,14 +33,132 @@ include::snippets/logging-stable-updates-snip.adoc[]
 +
 The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it is created for you.
 
-. Select *Enable operator-recommended cluster monitoring on this namespace.*
+. Select *Enable Operator-recommended cluster monitoring on this namespace.*
 +
 This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
 
 . For *Update approval* select *Automatic*, then click *Install*.
 +
 If the approval strategy in the subscription is set to *Automatic*, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to *Manual*, you must manually approve pending updates.
 
+. Install the Red{nbsp}Hat OpenShift Logging Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *Red{nbsp}Hat OpenShift Logging* from the list of available Operators, and click *Install*.
+
+.. Ensure that the *A specific namespace on the cluster* is selected under *Installation Mode*.
+
+.. Ensure that *Operator recommended namespace* is *openshift-logging* under *Installed Namespace*.
+
+.. Select *Enable Operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-logging` namespace.
+
+.. Select *stable-5.y* as the *Update Channel*.
```

Comment:
In the CLI install procedure, the user could specify [either `stable` or `stable-5.y`](https://github.com/openshift/openshift-docs/pull/79266/files#diff-68fcd41a52a46ad15113fa745cff38e770f3994daa910e3c42eafc4ce3eefb67R64). Is this a discrepancy?